src/Amphitheatre/common.jl

```diff
@@ -0,0 +1,131 @@
+
+abstract type AbstractPointPose end
+#NOTE names should match soft type names with Arena added to the front to avoid conflict, eg. ArenaPose2 and ArenaPoint2
+
+mutable struct ArenaPose2{T<:AbstractFloat} <: AbstractPointPose
+	x::T
+	y::T
+	θ::T
+end
+
+mutable struct ArenaPoint2{T<:AbstractFloat} <: AbstractPointPose
+	x::T
+	y::T
+end
+
+mutable struct ArenaPoint3{T<:AbstractFloat} <: AbstractPointPose
+	x::T
+	y::T
+	z::T
+end
+
+"""
+    $(SIGNATURES)
+Set the draw transform for the meshcat root object.
+
+Example:
+```julia
+# default is Z up
+# make Z down
+setGlobalDrawTransform!(vis, quat=Quat(0.0,1.0,0.0,0.0))
+```
+"""
+function setGlobalDrawTransform!(vis::Visualizer;trans=Translation(0.0,0.0,0.0), quat::Rotations.Quat=Quat(1.0,0.0,0.0,0.0))
```

Comment:
I think we should merge this first, and then in a separate PR try concentrate these functions `setGlobal...`, `visPose!`, etc with https://github.com/JuliaRobotics/Arena.jl/blob/ce719fbeae821040cac7c57cfc55568df9a02943/src/VisualizePosesPoints.jl#L5

Reply:
similar to what you did with depthclouds.  Move the common stuff into src/common.

Reply:
Very hard to do currently without changing existing functions in arena, this is one of the reasons I made a sub module. I agree, we can focus on getting the common components in the future.